### PR TITLE
Update offline Ansible deployment

### PR DESCRIPTION
**Pull request by lilin90**
Via https://github.com/pingcap/docs-cn/pull/494.

### CODE REVIEW

File: op-guide/offline-ansible-deployment.md [hunk 1]
```diff
@@ -260,13 +243,7 @@ Descriptions about the two circumstances are as follows.
...
         ```
     
-    2. Connect to the network and download TiDB binary to the Control Machine.
-
-        ```
-        ansible-playbook local_prepare.yml
-        ```
-    
-    3. Initialize the system environment and edit the kernel parameters.
+    2. Initialize the system environment and edit the kernel parameters.
```
Comment (CaitinChen) [inline op-guide/offline-ansible-deployment.md]: Item number 4？

File: op-guide/offline-ansible-deployment.md [hunk 2]
```diff
@@ -280,13 +257,13 @@ Descriptions about the two circumstances are as follows.
         ansible-playbook bootstrap.yml -k
         ```
     
-    4. Deploy the TiDB cluster.
+    3. Deploy the TiDB cluster.
```
Comment (CaitinChen) [inline op-guide/offline-ansible-deployment.md]: 5？

File: op-guide/offline-ansible-deployment.md [hunk 3]
```diff
@@ -280,13 +257,13 @@ Descriptions about the two circumstances are as follows.
...
 
         ```
         ansible-playbook deploy.yml -k
         ```
     
-    5. Start the TiDB cluster.
+    4. Start the TiDB cluster.
```
Comment (CaitinChen) [inline op-guide/offline-ansible-deployment.md]: 6？

File: op-guide/offline-ansible-deployment.md (hunk 1 above)
Comment (lilin90) [reply op-guide/offline-ansible-deployment.md]: Just leave it as "2". Please pay attention to the line number on the left.

**Comment by lilin90** [comment]
@liubo0127 Technical review

**Pull request merged**